Cargo.toml

```diff
@@ -72,7 +72,7 @@ version = "=1.10.0"
 
 [dependencies.leo-errors]
 path = "./errors"
-version = "=1.10.0"
+version = "=1.10.1"
```

Comment:
Why does this version need to update?
Ditto for the rest

Reply:
I added a new error, but couldn't compile unless I updated the version. Or is there another way to fix this?

Reply:
If you add your errors to the end of the file (as you did), it will remain backwards compatible and there should be no need to update the version.
These versions should only be updated on a new release.

Reply:
@d0cd Hi sir, I have reverted those updated versions